XTS/WS-C/dev/src/com/arjuna/webservices11/wscoor/client/WSCOORClient.java

```diff
@@ -94,7 +100,25 @@ public static RegistrationPortType getRegistrationPort(final W3CEndpointReferenc
          */
 
         Map<String, Object> requestContext = bindingProvider.getRequestContext();
-        MAP map = AddressingHelper.outboundMap(requestContext);
+        MAP map;
```

Comment:
this is about my feeling: there is quite a lot a new code in comparison with the old one. What about considering to have some method with all this logic?